pkg/auth/auth_handler.go

```diff
@@ -219,6 +220,7 @@ func (a *AuthHandler) CreateAuthenticator() func(name, in string, authenticate s
 			}
 			p, err := authenticate(token)
 			if err != nil {
+				a.log.Errorf("Fail to authenticate: Req_id %s, Error %v", requestid.FromContext(r.Context()), err)
```

Comment:
just to be in a standard suggesting to create a new logger as any other place:
```
log := logutil.FromContext(r.Context(), a.log)
```
Then you will not need to write the request id manually and work as any other place.

Reply:
Done